config.json

```diff
@@ -811,7 +811,11 @@
       "core": false,
       "difficulty": 7,
       "slug": "simple-linked-list",
-      "topics": null,
+      "topics": [
+        "algorithms",
+        "generics",
+        "lists"
```

Comment:
Great suggestions! I would also add `exception_handling` since the tests expect an exception to be thrown :)